.travis.yml

```diff
@@ -19,6 +20,9 @@ install:
 script:
   - pytest
 
+after_success:
+  - codecov
```

Comment:
It is easier to use codecov-bash.
Does not require to pip-install codecov then.